web/server.js

```diff
@@ -261,6 +261,17 @@ module.exports = function(options) {
           reply(json.user);
         });
       }
+    },
+    {
+      method: 'POST',
+      path: '/logout',
+      config: {
+        auth: false
```

Comment:
I suspect part of the reason that some web frameworks put so little security around logout endpoints is that logging out generally always results in the same or less privileges than the user had before, so it's never a threat, and should always be easy to do? I mean, at most it seems like you could annoy someone by constantly logging them out of a third-party site without their permission, but that seems like the extent of the damage?

I have no idea.